api/v1/controllers/samples.js

```diff
@@ -88,7 +88,7 @@ module.exports = {
       const sampleName = req.swagger.params.key.value.toLowerCase();
 
       redisModelSample.getSample(
-        sampleName, resultObj, res.method, req.swagger.params
+        sampleName, resultObj, req.method, req.swagger.params
```

Comment:
what is the reason behind changing the response.method to request.method? Are they identical?